### PR TITLE
[feat-5236]: Images in my incidents with and without subtitle

### PR DESCRIPTION
**Pull request by OscarBakker**
Add a more descriptive html structure first. 

Then add captions and figures. Separate attachments created by municipality and by user.

Ticket: [SIG-5236](https://gemeente-amsterdam.atlassian.net/browse/SIG-5236)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/my-incidents/components/IncidentsDetail/styled.ts [hunk 1]
```diff
@@ -9,35 +9,65 @@ export const StyledBacklink = styled(BackLink)`
...
   margin: ${themeSpacing(0, 0, 1)};
 `
 
+export const DescriptionWrapper = styled.div`
+  margin-bottom: ${themeSpacing(6)};
+`
+
+export const StyledDD = styled.dd`
+  font-weight: 700;
+  font-size: 18px;
```
Comment (vdegraaf) [inline src/signals/my-incidents/components/IncidentsDetail/styled.ts:23]: should be rem

File: src/signals/my-incidents/components/IncidentsDetail/styled.ts [hunk 2]
```diff
@@ -9,35 +9,65 @@ export const StyledBacklink = styled(BackLink)`
...
+`
+
+export const StyledDL = styled.dl``
```
Comment (vdegraaf) [inline src/signals/my-incidents/components/IncidentsDetail/styled.ts:26]: not very much styling :p

File: src/signals/my-incidents/components/IncidentsDetail/styled.ts [hunk 3]
```diff
@@ -9,35 +9,65 @@ export const StyledBacklink = styled(BackLink)`
...
   object-fit: cover;
   width: 100%;
-  height: 100%;
 `
 
-export const ImageWrapper = styled.div`
+export const StyledFigCaption = styled.figcaption``
+
+export const ImagesWrapper = styled.div`
+  gap: 8px;
```
Comment (vdegraaf) [inline src/signals/my-incidents/components/IncidentsDetail/styled.ts:36]: themeSpacing(2)

File: src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx [hunk 4]
```diff
@@ -29,6 +32,14 @@ export const IncidentsDetail = ({
...
   const attachments = _links?.['sia:attachments']
 
+  const attachmentsUser = attachments?.filter(
+    (attachment) => !attachment.created_by
+  )
+
+  const attachmentsMunicipality = attachments?.filter(
+    (attachment) => attachment.created_by
+  )
+
```
Comment (vdegraaf) [inline src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx:42]: is `created_by` the best way to distinguish between these two? Attachments added by the user never have a created_at?

**Review by vdegraaf — COMMENTED** [review]
Nice. Just some small questions/comments

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]
- [ ] Toen ik foto's had ge-upload als melder, kreeg ik deze melding: Capture Vervolgens zie je bij melding 13117 wel alle foto's die ik als melder heb ge-upload, maar in mijn-meldingen zie je de Jebediahfoto niet. Lijkt me dat we alle foto's die in de backoffice zien, ook zouden moeten zien in mijn-meldingen (als ze van de melder zijn)

File: src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.test.tsx [hunk 5]
```diff
@@ -65,7 +65,33 @@ describe('IncidentsDetail', () => {
...
     )
 
-    expect(screen.getByText("Foto's")).toBeInTheDocument()
+    expect(screen.getByText(/Foto's/)).toBeInTheDocument()
+
+    const attachmentMunicipality = incidentsDetail._links[
+      'sia:attachments'
+    ].map((attachment) => ({
+      ...attachment,
+      created_by: 'Gemeente Amsterdam',
```
Comment (SireeKoolenWijkstra) [inline src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.test.tsx:74]: If the creator is an officer of the municipality, then the created_by should be empty. If you don't mean the officer, then maybe pick a different name? It's kinda confusing now (even though it's just a test)

**Review by vdegraaf — COMMENTED** [review]
Styling:

- photo's are no aligned as designed: 
Screenshot 2023-09-13 at 09 25 33

- Line-height of description should be 26px

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]
- [ ] There is no spacing between the picture in mijn-meldingen.

File: src/signals/my-incidents/types.ts [hunk 6]
```diff
@@ -1,18 +1,20 @@
...
 export interface MyIncidentsValue {
-  email?: string
   setEmail: (email: string) => void
-  incidentsList?: MyIncident[]
   setIncidentsList: (incidentsList: MyIncident[]) => void
+  incidentsList?: MyIncident[]
+  email?: string
 }
 
 interface Attachment {
```
Comment (SireeKoolenWijkstra) [inline src/signals/my-incidents/types.ts:13]: Isn't this also a case of "There already is an Attachment type. Extract that instead"? Incidentdetail of incident-management also has its own type of Attachment.

File: src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.test.tsx (hunk 5 above)
Comment (OscarBakker) [reply src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.test.tsx:74]: changed

File: src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx (hunk 4 above)
Comment (OscarBakker) [reply src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx:42]: yep, its distinguishing the two by the user and municipality

**Review by SireeKoolenWijkstra — COMMENTED** [review]
lgtm

**Review by SireeKoolenWijkstra — APPROVED** [review]

**Review by vdegraaf — APPROVED** [review]

**Pull request merged**